tests/unit/components/sl-chart-test.js

```diff
@@ -0,0 +1,121 @@
+import Ember from 'ember';
+import { moduleForComponent, test } from 'ember-qunit';
+
+var object = {
+    chartOptions: {
+        chart: {
+            type: 'bar'
+        },
+
+        xAxis: {
+            categories: [ 'Apples', 'Bananas', 'Oranges' ]
+        },
+
+        yAxis: {
+            title: {
+                text: 'Fruit Eaten'
+            }
+        }
+    }
+};
+
+var series = Ember.A([
```

Comment:
Do not use `Ember.A()`, only `[]` in this situation.  The reason for this is because the code under test only needs an array - not an Ember Array specifically.  Aim to keep the contrived test scenarios as basic and pure as possible so that we limit the amount of things they can be affected by and when a test fails it should be very obvious as to why it did and not because we set the test up incorrectly by introducing unneeded "overhead".